components/Navbar.js

```diff
@@ -28,14 +150,15 @@ export default function Navbar() {
             </StyledLink>
           </StyledDiv>
 
-          <StyledLink href="">
+          <StyledButton onClick={onLoginHandler}>
             <Image
               alt="metamask"
-              src={Metamask}
+              src={isConnected ? MetamaskActive : Metamask}
               width="100px"
               height="100px"
             ></Image>
-          </StyledLink>
+          </StyledButton>
+
           <StyledDiv active={pathname === "/tasks" ? true : false}>
             <StyledLink href="/tasks">
               <Image alt="tasks" src={Tasks} width="50px" height="50px"></Image>
```

Comment:
It is recommend to write your components as self closing tag:

```suggestion
              <Image alt="tasks" src={Tasks} width="50px" height="50px" />
```

Also change this for your other components without content.

Reply:
done!